ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -1340,7 +1340,7 @@ static bool ggml_vk_matmul_shmem_support(const vk_device& device, const std::vec
     // Needs to be kept up to date on shader changes
     const uint32_t bank_conflict_offset = device->coopmat_support ? 8 : 1;
     const uint32_t type_size = device->fp16 ? sizeof(ggml_fp16_t) : sizeof(float);
-    const uint32_t warps = warptile[0] / device->subgroup_size;
+    const uint32_t warps = warptile[0] / warptile[10];
```

Comment:
It has no effect if the warp size in the warptile is equivalent to `device->subgroup_size`. The reason I made this change was so that you could modify the warp size in the warptile without having to override line 1343 as well.